website/docs/guides/container_service_extension_4_0_cluster_management.html.markdown

```diff
@@ -0,0 +1,272 @@
+
+In order to complete the steps described in this guide, please be aware:
+
+* CSE v4.0 must be installed. Read the [installation guide][cse_install_guide] for more information.
+* Terraform provider needs to be v3.9.0 or above.
+* CSE Server must be up and running.
+
+## Creating a Kubernetes cluster
+
+-> You can have a look at a working example of a TKGm cluster [here][cluster]. It is encouraged to read the following
+section to understand how it works.
+
+To be able to create a TKGm cluster, one needs to prepare a [`vcd_rde`][rde] resource. In the [proposed example][cluster],
+this RDE is named `k8s_cluster_instance`. The important arguments to take into account are:
+
+- **`resolve` must be always `false`**, because the CSE Server is the responsible for performing the RDE resolution when the
+  TKGm cluster is completely provisioned, so Terraform should not interfere with this process.
+- **`resolve_on_removal` must be always `true`**, because the RDE is resolved by the CSE Server and not by Terraform. If one
+  wants to execute a Terraform destroy without the RDE being resolved, the operation will fail. Being `true` assures that Terraform
+  can perform a Terraform destroy in every case. See ["Deleting a Kubernetes cluster"](#deleting-a-kubernetes-cluster) section for more info.
+
+The [`vcd_rde`][rde] argument `input_entity` is taking the output of the Terraform built-in function `templatefile`, that references
+a JSON template that you can find [here][tkgmcluster_template]. This function will set the correct values to the following
+placeholders that can be found in that file:
+
+- `vcd_url`: The VCD URL, the same that was used during CSE installation.
+- `name`: This will be the TKGm cluster name. It must contain only lowercase alphanumeric characters or '-',
+  start with an alphabetic character, end with an alphanumeric, and contain at most 31 characters.
+- `org`: The Organization in which the TKGm clusters will be created. In this guide it was created as `tenant_org` and named
+  "Tenant Organization" during CSE installation phase.
+- `vdc`: The VDC in which the TKGm clusters will be created. In this guide it was created as `tenant_vdc` and named
+  "Tenant VDC" during CSE installation phase.
+- `capi_yaml`: This must be set with a valid single-lined CAPVCD YAML, that will be explained next.
+- `delete`: This is used to delete a cluster. See ["Deleting a Kubernetes cluster"](#deleting-a-kubernetes-cluster) section for more info.
+  During creation it should be always `false`.
+- `force_delete`: This is used to forcefully delete a cluster. See ["Deleting a Kubernetes cluster"](#deleting-a-kubernetes-cluster) section for more info.
+  During creation it should be always `false`.
+- `auto_repair_on_errors`: Setting this to `true` will make the CSE Server to repair the TKGm cluster on errors. You can change
+  this to `false` if you want to troubleshoot any error by yourself.
```

Comment:
This is a bit unclear, maybe we should remove the last sentence, as it should be given by the argument itself.
```suggestion
- `auto_repair_on_errors`: Setting this to `true` will make the CSE Server try to auto-repair the errors of the TKGm cluster.
```